elixir/apps/web/lib/web/live/settings/dns.ex

```diff
@@ -44,7 +44,6 @@ defmodule Web.Settings.DNS do
             target="_blank"
           >
             Read more about configuring DNS in Firezone.
-            <.icon name="hero-arrow-top-right-on-square" class="-ml-1 mb-3 w-3 h-3" />
```

Comment:
This icon was inconsistently applied to external links and doesn't mesh well with inline links so it was removed